### ISSUE TITLE
Xournal++ crashes while switching toolbar in portraitmode

### ISSUE THREAD

**Issue by ethpi** [xournalpp/xournalpp#4997]
**Affects versions :**
 - OS: Ubuntu 20.04
 - (Linux only) Desktop environment: KDE Plasma
 - xournalpp 1.2.0
└──libgtk: 3.24.20
 - Installation method: PPA

While switiching toolbar in protaitmode on, xournal++ crashes

1.`rm -r ~/.config/xournalpp`  _(changed from `rm ~/.config/.xournalpp`)_
2. start xournal++
3. click View / toolbar / portrait

[errorlog.20230803-100224.log](https://github.com/xournalpp/xournalpp/files/12248406/errorlog.20230803-100224.log)

**Comment by rolandlo** [comment]
There is no file/folder `~/.config/.xournalpp` in version 1.2.0. Do you mean the folder `~/.config/xournalpp` instead?

**Comment by rolandlo** [comment]
Ok, but it's a folder, do you use `rm -rf` instead?

**Comment by ethpi** [comment]
Same. Of course I used `rm -r `. I must be drunken. I will change the initial post once again. Sorry.

**Comment by rolandlo** [comment]
I think you are not supposed to delete that folder. And if you do, the crash only happens the first time you start Xournal++, the second time it's already working fine.

**Comment by ethpi** [comment]
Your are right. The second start is ok.

I only deleted this folder to start with a fresh xournal++ to have a reproducible setting. 

Background: In the first place I tried to report another crash that only appears with my old settings.xml. In this settings I connected the second button of my wacom stylus with the select-area-tool (in German: "Bereich auswählen"). When I press this second button a few mm above the tablet and touch afterwards the tablet, my xournal++ 1.2.0 crashes. When I do this with fresh settings xournal++ 1.2.0 doesn't crash.  It seems there is problem with my settings upgrading from 1.1 to 1.2.0 
But this is another story for another issue.
